protos/camera/camera.proto

```diff
@@ -190,7 +190,7 @@ message ModeResponse {
 
 message SubscribeVideoStreamInfoRequest {}
 message VideoStreamInfoResponse {
-    VideoStreamInfo video_stream_info = 1; // Video stream info
+    repeated VideoStreamInfo video_stream_infos = 1; // Video stream infos
```

Comment:
How does this work when it's repeated? Do you get multiple video stream infos, one per video stream?